src/main/java/org/example/models/User.java

```diff
@@ -0,0 +1,44 @@
+package org.example.models;
+
+public class User {
+
+    String username;
+
+    String password;
+
+    int loginID;
+
+    public User() {
+    }
+
+    public User(final String username, final String password,
+                final int loginID) {
+        this.username = username;
+        this.password = password;
+        this.loginID = loginID;
```

Comment:
what is loginID and how does it differ from username?

Reply:
i have renamed this to UserRoleID, each UserRole (user, admin, applicant etc) will have a unique ID that is referenced in the user table